assets/smart-grid.scss

```diff
@@ -0,0 +1,166 @@
+@use "sass:math";
```

Comment:
Can you explain why do we need this?

Reply:
These styles are used in vuestic landing, had to bring them here to have matching popups

Reply:
These are kinda deprecated right now.

Maybe we can at least comment it's deprecated?